### PR TITLE
fix(cancel-order): fix order cancellation error message

### LINKED ISSUE #2214: Incorrect error message when cancel already filled order
**Description**

Reported in https://github.com/cowprotocol/cowswap/pull/2187#issuecomment-1488794089

'Filled' notifications come with a delay, so sometimes when an order is already filled, and a user tries to cancel it, this notification appear. 
**To reproduce:** 

1. Place an order
2. Call 'cancel order modal, send a TX request to the wallet, but do not sign it until you hear the Moo sound
3. Immediately sign the TX in the wallet. 

**AR:** 
image

**Expected behavior**
image

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Fixes #2214

image

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]
Great!

File: src/cow-react/common/hooks/useCancelOrder/index.ts [hunk 1]
```diff
@@ -47,12 +46,12 @@ export function useCancelOrder(): (order: Order) => UseCancelOrderReturn {
...
       // Check the 'cancellability'
 
-      const isEthFlowOrder = getIsEthFlowOrder(order)
+      // const isEthFlowOrder = getIsEthFlowOrder(order)
 
       // 1. EthFlow orders will never be able to be cancelled offChain
       // 2. The wallet must support offChain singing
       // 3. The order must be PENDING
-      const isOffChainCancellable = !isEthFlowOrder && allowsOffchainSigning && order?.status === OrderStatus.PENDING
+      const isOffChainCancellable = true
```
Comment (anxolin) [inline src/cow-react/common/hooks/useCancelOrder/index.ts:54]: @shoom3301 ❓❓❓
i don't get this change

Comment (shoom3301) [reply src/cow-react/common/hooks/useCancelOrder/index.ts:54]: I've tested with it, my bad.
https://github.com/cowprotocol/cowswap/issues/2392 - there is a fix

Comment (anxolin) [reply src/cow-react/common/hooks/useCancelOrder/index.ts:54]: Ahh, ok it was a test

**Pull request merged**